src/libs/API/types.ts

```diff
@@ -388,6 +389,7 @@ type WriteCommandParameters = {
     [WRITE_COMMANDS.UPDATE_POLICY_DISTANCE_RATE_VALUE]: Parameters.UpdatePolicyDistanceRateValueParams;
     [WRITE_COMMANDS.SET_POLICY_DISTANCE_RATES_ENABLED]: Parameters.SetPolicyDistanceRatesEnabledParams;
     [WRITE_COMMANDS.DELETE_POLICY_DISTANCE_RATES]: Parameters.DeletePolicyDistanceRatesParams;
+    [WRITE_COMMANDS.UPDATE_CONNECTION_CONFIG]: Parameters.UpdateConnectionConfigParams;
```

Comment:
```suggestion
    [WRITE_COMMANDS.UPDATE_POLICY_CONNECTION_CONFIG]: Parameters.UpdatePolicyConnectionConfigParams;
```

I think it is better if we include "policy" in the name of these variables/types so we know that it is about connections belonging to policies

Reply:
updated https://github.com/Expensify/App/pull/38006/commits/14ae20a2d427e1f360d7638ef804685aeb797624